### ISSUE TITLE
Github Pages website not displaying

### ISSUE THREAD

**Issue by Tesla-Atoz** [mldangelo/personal-site#268]
I removed the earlier build folder and ran npm install.
Before that , I had my Homepage set as per adapting.md . I ran npm run deploy and it says the site is published, then i go the link from settings-> githubpages link . then  its showing a 404 error page.But the  site is working fine in localhost. Can anyone tell me where I am dng wrong. I need this site to be deployed and working asap. @mldangelo  thnx for your site and allowiing us to use it

![Screenshot (32)](https://user-images.githubusercontent.com/53090203/106311852-c9c46680-628b-11eb-8639-ca50930ba796.png)
.

**Comment by mldangelo** [comment]
Hi @Tesla-Atoz I'm sorry for my delay in helping you resolve your issues. I read through your repository (https://github.com/Tesla-Atoz/Dinesh-Pandikona) and think that your issue is due to how your set up your domain redirect. You should try adding a CNAME file to point your new domain. Read https://docs.github.com/en/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site for more information about setting it up. In the meantime, I'm going to try to replicate your environment. Please feel free to send me an email if you'd like to debug via video chat.

**Comment by Tesla-Atoz** [comment]
Yeah. I totally forgot about it. It's my custom domain name. I will try to
add cname file and try to deploy.

But I tried to deploy to netlify, in this case I tried through git
continuous integration on netlify. It's getting deployed but displaying a
white web page.

Am i missing something?

Anyhow, thanks for reply and for your open source site.

**Comment by Tesla-Atoz** [comment]
Hello @mldangelo I added the CNAME file and ran npm run deploy after pushing. The website is deployed and the css of website is not working. 
Below is the preview of what it looks after deployment. 
![no_css_web_view_after_deployment](https://user-images.githubusercontent.com/53090203/106419540-b0cfd700-647e-11eb-824b-0ddcb4acb911.png)


Hey @mldangelo  if i solve this problem then it would be fine that my portfolio will be live just like yours. Expecting a quick reply.

**Comment by mldangelo** [comment]
Hi @Tesla-Atoz, take a quick look at your deployed source: `view-source:https://vegmedquotes.xyz/dineshpandikona/`. 

You'll notice that your css path is `/Dinesh-Pandikona/static/css/main.28570286.chunk.css`. Your `homepage` in package.json is correct, so it looks like you need to run `npm run deploy` one more time, which should set the correct CSS path. 

I hope this helps. Re. netlify, I don't officially support it, but I'll take a look take a look this week.

**Comment by Tesla-Atoz** [comment]
Thanks @mldangelo  its deployed. I just ran n`pm run deploy` it got published with nice css. here is  quick view.

[](https://vegmedquotes.xyz/dineshpandikona/)

Michael, I just wanted to let you know that I am willing to contribute to this project. I worked out this project on windows machine. Just let me know if you want me to try something for this project.  :heart: :thumbsup:

A BIG THANKS BROTHER :smiley:
